examples/client/Locomotion/src/context/index.js

```diff
@@ -1,4 +1,5 @@
 export * from './main';
 export { RideStateContextContext, default as RideStateContextContextProvider } from './ridePageStateContext';
 export { RidePageContext, default as RidePageContextProvider } from './ridePageContext';
+export { RidePageContext as MewRidePageContext, default as NewRidePageContextProvider } from './newRideContext';
```

Comment:
i think we can just remove the old ride context instead of making every import as newcontext. i dont think the old one is used anywhere anymore